backend/test/services/release.spec.ts

```diff
@@ -0,0 +1,120 @@
+import { describe, expect, test, vi } from 'vitest'
+
+import { createRelease, deleteRelease, getModelReleases, getReleaseBySemver } from '../../src/services/v2/release.js'
+
+const arrayAsyncFilter = vi.hoisted(() => ({
+  asyncFilter: vi.fn(() => []),
+}))
+vi.mock('../../src/utils/v2/array.js', () => arrayAsyncFilter)
+
+const authorisationMocks = vi.hoisted(() => ({
+  userReleaseAction: vi.fn(() => true),
+}))
+vi.mock('../../src/connectors/v2/authorisation/index.js', async () => ({
+  ...((await vi.importActual('../../src/connectors/v2/authorisation/index.js')) as object),
```

Comment:
Why the need for actual implementation here?

Reply:
I don't want to mock the authorisation enums `ModelAction` and `ReleaseAction`.